File: Python/Program/TimEsport.py
# File: TimEsport.py
from AnggotaTim import AnggotaTim

class TimEsport:
    def __init__(self, nama_tim="", divisi_game=""):
        self._nama_tim = nama_tim
        self._divisi_game = divisi_game
        self._daftar_anggota = []

    def add_anggota(self, anggota):
        self._daftar_anggota.append(anggota)
        
    def get_nama_tim(self): return self._nama_tim
    def get_divisi_game(self): return self._divisi_game
    def get_daftar_anggota(self): return self._daftar_anggota

File: Java/Program/TimEsport.java
import java.util.ArrayList;

public class TimEsport {
    private String namaTim;
    private String divisiGame;
    private ArrayList<AnggotaTim> daftarAnggota;

    public TimEsport(String nama, String divisi) {
        this.namaTim = nama;
        this.divisiGame = divisi;
        this.daftarAnggota = new ArrayList<>();
    }

    public void addAnggota(AnggotaTim anggota) {
        this.daftarAnggota.add(anggota);
    }

    public String getNamaTim() { return this.namaTim; }
    public String getDivisiGame() { return this.divisiGame; }
    public ArrayList<AnggotaTim> getDaftarAnggota() { return this.daftarAnggota; }
}